### PR TITLE
Create data model with migrations

### PR DESCRIPTION
**Pull request by tkz96**
This PR is to:

- Generate database schema with migrations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
```
Comment (t-yanick) [inline README.md]: - [ ] Kindly update your readme file to reflect the changes for this milestone, checkout this [template](https://github.com/microverseinc/readme-template) from Microverse and kindly adapt it to your project.

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @tkz96, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments below and under the review._ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

#### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220901150432_create_user.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```
Comment (ahmetbozaci) [inline db/migrate/20220901150432_create_user.rb]: ### OPTIONAL

 _I make it optional but it is a requirement that is not mentioned in the last review. Kindly fix it._

- [ ] Bio type should be text. Kindly change it.

File: db/migrate/20220901151735_create_post.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.string :text
```
Comment (ahmetbozaci) [inline db/migrate/20220901151735_create_post.rb]: ### OPTIONAL
_I make it optional but it is a requirement that is not mentioned in the last review. Kindly fix it._

- [ ] Text type should be text. Kindly change it.

File: db/migrate/20220901151735_create_post.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.references :users, :post, foreign_key: true
```
Comment (ahmetbozaci) [inline db/migrate/20220901151735_create_post.rb]: ### OPTIONAL
_I make it optional but it is a requirement that is not mentioned in the last review. Kindly fix it._
- [ ] References name should be an author.

image

File: db/migrate/20220901151743_create_comment.rb [hunk 5]
```diff
@@ -0,0 +1,11 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :users, null: false, foreign_key: true
```
Comment (ahmetbozaci) [inline db/migrate/20220901151743_create_comment.rb]: ### OPTIONAL
_I make it optional but it is a requirement that is not mentioned in the last review. Kindly fix it._
- [ ] References name should be an author.

image

File: db/migrate/20220901151752_create_like.rb [hunk 6]
```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :users, null: false, foreign_key: true
```
Comment (ahmetbozaci) [inline db/migrate/20220901151752_create_like.rb]: ### OPTIONAL
_I make it optional but it is a requirement that is not mentioned in the last review. Kindly fix it._
- [ ] References name should be an author.

image

**Review by ahmetbozaci — APPROVED** [review]
# 🥳 🎉🥇 💯  💥 🎆 🎈 STATUS: APPROVED 🥳 🎉  🥇  💯  💥   🎆 🎈 🎉 



@tkz96   ✋🏼 
#### Your project is great. 🎉 I approve of it. There is nothing else to say other than... it's time to merge it. You can submit this project on your dashboard.
___ 

### 💯 Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

___ 

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
aproved

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._
___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Pull request merged**